cmake/Common.cmake

```diff
@@ -15,10 +15,11 @@ function(package_set_postinst_prerm LIB_NAMES LIB_DIRS INCLUDE_DIRS)
         list(GET LIB_NAMES ${val} lib_name)
         list(GET LIB_DIRS  ${val} lib_dir)
         list(GET INCLUDE_DIRS ${val} inc_dir)
+        list(GET LDCONF_ENTRYS ${val} ldconf_entry)
 
         set(POSTINST_SOURCE "${POSTINST_SOURCE}\nmkdir -p ${inc_dir}/../../include/")
         set(POSTINST_SOURCE "${POSTINST_SOURCE}\nmkdir -p ${lib_dir}/../../lib/cmake/${lib_name}")
-        set(POSTINST_SOURCE "${POSTINST_SOURCE}\necho \"${lib_dir}\" > /etc/ld.so.conf.d/${lib_name}.conf")
+        set(POSTINST_SOURCE "${POSTINST_SOURCE}\necho \"${ldconf_entry}\" > /etc/ld.so.conf.d/${lib_name}.conf")
```

Comment:
I need time to review this change.

Reply:
The changes are mainly to add a new parameter and this parameters adds the ld.so.conf entry which is passed as /opt/rocm.